addons/html_editor/static/src/main/font/font_plugin.js

```diff
@@ -270,8 +270,9 @@ export class FontPlugin extends Plugin {
             this.handleSplitBlockPRE.bind(this),
             this.handleSplitBlockquote.bind(this),
         ],
-        delete_backward_overrides: withSequence(20, this.handleDeleteBackward.bind(this)),
-        delete_backward_word_overrides: this.handleDeleteBackward.bind(this),
+
+        /** Predicates */
+        empty_blocks_to_base_container_predicates: (node) => handledTags.includes(node.nodeName),
```

Comment:
`empty_base_container_candidates_predicates` ? What do you think @abd-msyukyu-odoo ?